baselines/dom.generated.d.ts

```diff
@@ -15360,6 +15414,8 @@ type ScrollRestoration = "auto" | "manual";
 type FormDataEntryValue = string | File;
 type InsertPosition = "beforebegin" | "afterbegin" | "beforeend" | "afterend";
 type HeadersInit = Headers | string[][] | { [key: string]: string };
+type DOMString = string;
+type AnimationKeyFrame = {[key: string]: string | number | [string | number, string | number] | undefined};
```

Comment:
the spec does not say it has to be a tuple.. there are examples https://drafts.csswg.org/web-animations/#example-be6aa998 where it is an array of strings.

Reply:
`null` seems to be allowed as well

Reply:
Done: I defined the type
```typescript
{offset?: number | null | (number | null)[]} & {[key: string]: string | number | number[] | string[]};
```